pkg/provisioner/ironic/ironic.go

```diff
@@ -45,6 +45,9 @@ const (
 )
 
 func init() {
+	// FIXME
+	return
```

Comment:
Maybe we could move out GetHardwareDetails to utils, or someone bypass all this environment variable checking. I'd like to be able to reuse some functions from here

Reply:
I'd support moving GetHardwareDetails somewhere else.

Reply:
Maybe we could put the hardware details stuff in a pkg/provisioner/ironic/hardwaredetails or something so it can be reused but is still clearly part of the ironic provisioner code tree.

Reply:
I think the approach we want to take though is that we'll just support an annotation for the raw inspection data (similar to 470), to make what we're trying to do (gather inspection out of band for externally provisioned hosts) as simple as `curl ... | kubectl patch ....`

Is this tool still useful to someone?